charts/replicated-library/templates/lib/_pod_annotations.tpl

```diff
@@ -18,6 +18,69 @@
     {{- end -}}
   {{- end -}}
   {{- if $configMapsFound -}}
-    {{- printf "checksum/config: %v" (include ("replicated-library.configmaps") . | sha256sum) | nindent 0 -}}
+    {{- include ("replicated-library.podAnnotations.shaAnnotations") . -}}
```

Comment:
We should probably add an override that allows people to turn this off entirely. While it's not as common, applications like HAProxy do live reloads and you wouldn't want to restart the application in that case.